Resources/views/app/base.html.twig

```diff
@@ -1,8 +1,15 @@
 {# Resources/views/app/base.html.twig #}
 {% extends '@App/base.html.twig' %}
 
+{% set meta_title = kimai_config.get('aarhuskommune.meta-title') %}
+
 {% block body_start %}
-     class="{{ app.request.attributes.get('_route')|slug|lower }}" data-title="AAK-tid – {{ block('page_title')|e('html_attr') }}"
+    class="{{ app.request.attributes.get('_route')|slug|lower }}"
+    {% if meta_title is not empty %}
+        data-title="{{ meta_title|raw }}"
+    {% else %}
+        {{ parent() }}
```

Comment:
This will render all attributes the parent block renders, i.e. we will potentially get the `class` attribute twice. Can we safely assume that the (parent) `body_start` block does not render a `class` attributes?